### PR TITLE
Dotnet supported 3.1

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

**Review by dinaabennett — APPROVED** [review]

File: modules/ROOT/pages/_partials/supported-versions.adoc [hunk 1]
```diff
@@ -152,12 +152,20 @@ a| Windows 10 +
...
+|14.2
+
+|.NET Android
+|6.0 
 |5.1/API 22
 
 |===
 
-NOTE: Support for API 19, API 20and API 21 is deprecated and will be removed in a future release; you should plan to migrate as soon as possible.
+NOTE: Support for API 19, API 20 and API 21 is deprecated and will be removed in a future release; you should plan to migrate as soon as possible.
```
Comment (borrrden) [inline modules/ROOT/pages/_partials/supported-versions.adoc]: This is no longer needed as API 22 is the min version listed above.

**Review by borrrden — CHANGES_REQUESTED** [review]

**Comment by osfameron** [comment]
Addressed one change from @borrrden, can I get a re-review to merge?

**Review by dinaabennett — APPROVED** [review]
:-)

**Review by borrrden — APPROVED** [review]
I am pretty sure I will need another change later but I will file another PR at that time.

**Pull request merged**